src/neo/Ledger/Blockchain.cs

```diff
@@ -61,8 +61,7 @@ private class ParallelVerified { public Transaction Transaction; public bool Sho
         private readonly NeoSystem system;
         private readonly List<UInt256> header_index = new List<UInt256>();
         private uint stored_header_count = 0;
-        private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
```

Comment:
Why this is removed?

Reply:
`block_cache` is used to store the correct blocks (block.Index > Height + 1) that have been verified by headers. 
Now we don't synchronize the block headers, so we can't verify that the blocks are correct,.So they are all stored in `unverified_blocks_candidates`.

Reply:
But `TaskManager` is not removed, isn't it?

Reply:
Now `TaskManager` is used to sync `Tx` and `Consensus` message.

Reply:
SyncManager only focus on syncing block by block index.
TaskManager only focus on inv task.